metadata.rb

```diff
@@ -3,16 +3,15 @@
 maintainer_email 'help@sous-chefs.org'
 license          'Apache 2.0'
 description      'Installs/Configures samba'
-version          '0.13.0'
-
-recipe 'samba::default', 'Includes the samba::client recipe'
-recipe 'samba::client', 'Installs smbclient package'
-recipe 'samba::server', 'Installs samba server packages and configures smb.conf'
+version          '1.0.0'
 
 %w( debian ubuntu centos fedora redhat scientific amazon oracle ).each do |os|
   supports os
 end
 
-source_url 'https://github.com/sous-chefs/samba' if respond_to?(:source_url)
-issues_url 'https://github.com/sous-chefs/samba/issues' if respond_to?(:issues_url)
-chef_version '>= 11.0' if respond_to?(:chef_version)
+source_url 'https://github.com/sous-chefs/samba'
+issues_url 'https://github.com/sous-chefs/samba/issues'
+chef_version '>= 12.1' if respond_to?(:chef_version)
+
+depends 'compat_resource', '>= 12.16.3'
+depends 'yum'
```

Comment:
Do we actually need this anymore? Seems like we can just move this to berksfile for the testing stuff